### ISSUE TITLE
bug report: teamsAppExtendToM365.UnhandledError

### ISSUE THREAD

**Issue by Internet-of-Bayer** [OfficeDev/TeamsFx#8928]
An unexpected error has occurred while performing the <REDACTED: user-file-path> task. The reason for this error is: Unexpected token '', "mX-l��Q�"... is not valid JSON. Welcome to report this issue by clicking on the provided "Issue Link", so that we can investigate and resolve the problem as soon as possible.

stack:
SyntaxError: Unexpected token '', "mX-l��Q�"... is not valid JSON
	at JSON.parse (<anonymous>)
	at ConvertTokenToJson (<REDACTED: user-file-path>:8:5763866)
	at CodeFlowLogin.<anonymous> (<REDACTED: user-file-path>:8:5756089)
	at Generator.next (<anonymous>)
	at fulfilled (<REDACTED: user-file-path>:8:5750045)

**Comment by kuojianlu** [comment]
@Internet-of-Bayer 
This is a known issue related to login. Followings are the workaround that may unblock you.

Sign out the M365 account, and then re-sign in. After that, retry F5.
If the above method cannot solve this error, comment out teamsApp/extendToM365 action in teamsapp.local.yml. After that, retry F5. (Please note that this workaround is not applicable to the scenario that you debug in Outlook or the Microsoft 365 app)
![image](https://user-images.githubusercontent.com/37978464/240117238-e56087fe-a5c1-4351-86cf-135d5c1a8c9a.png)

**Comment by kuojianlu** [comment]
Duplicate of https://github.com/OfficeDev/TeamsFx/issues/8741.
